### PR TITLE
Release 3.3.0

### PR DESCRIPTION
**Pull request by renatocaval**
Updates the website with the lasted release. 

I'm in doubt to what we should mention. What are the binary compatibility changes we have? Anything to mention for migration?

### CODE REVIEW

File: news/_posts/2019-01-30-slick-3.3.0-released.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+, [matthiassig12](https://github.com/matthiassig12), [smootoo](https://github.com/smootoo)
+, [stefanholzmueller](https://github.com/stefanholzmueller), [sullis](https://github.com/sullis)
+, [trevorsibanda](https://github.com/trevorsibanda)
+and [xuwei-k](https://github.com/xuwei-k)
+
+##### These are the major changes in Slick 3.3.0 (since 3.2.3):
+
+* [``381d6f60``](https://github.com/slick/slick/commit/381d6f606e05e35273482bca95775af74c7cd550) improved TypePattern regex
+* [``5fab5553``](https://github.com/slick/slick/commit/5fab5553142e15a2548f9ad894fe36cf0a99db6f) improved string/char replacement
+* [``c5c48deb``](https://github.com/slick/slick/commit/c5c48deba1b1a34a25206cafa9b9ee270e2a7ef7) removed the _ in front of val names
```
[the remark this answers is not in the document]
Comment (renatocaval) [reply news/_posts/2019-01-30-slick-3.3.0-released.md]: I agree. We used to add lots of information in Play/Lagom releases and we decided to link to GitHub and only mention the most important change

File: download/index.md [hunk 2]
```diff
@@ -3,26 +3,26 @@ layout: doc
 title: Download
```
Comment (renatocaval) [inline download/index.md]: that whole section should be reviewed. 

Some examples still point to v3.1 and Activator is long gone. 

I will fix it in a second iteration.

File: news/_posts/2019-01-30-slick-3.3.0-released.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+layout: news
+title: Slick 3.3.0
+author: Renato Cavalcanti
+---
+We have just released Slick 3.3.0
```
Comment (renatocaval) [inline news/_posts/2019-01-30-slick-3.3.0-released.md]: I published exactly the same text here:

https://github.com/slick/slick/releases/tag/v3.3.0

**Comment by renatocaval** [comment]
Ok to merge this one?

**Comment by hvesalai** [comment]
Yes, do merge!

**Pull request merged**